P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -735,6 +735,46 @@ target="#CONA"/>).  The tag -->The
 
 
     </div>
+    
+    <div type="div3" xml:id="NDEVTN">
+      <head>Event Names</head>
+      <p><specList><specDesc key="eventName"/></specList></p>
+      
+      <p>The <gi>eventName</gi> element is provided in parallel to other naming mechanisms, and especially for <soCalled>canonical</soCalled>
+      names that are used in texts to mention specific named events.
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <p>On <date when="1719-03-19">Monday</date>, <rs type="person">she</rs> was writing about the 
```

Comment:
Does that general encoding with just the value "person" make sense? At least, the date and the two events in this example are encoded in a more detailed way, i.e. reference to something specific.

Reply:
Yes, agreed that the use of `<rs>` here may be a bit distracting, especially since it's not providing additional info...

Note that this is only instance of the use of `<rs>` tagging just a pronoun (i.e. the rest are slightly more specific, including things like "his lady" or "City of Brotherly Love") [NB: this example is duplicated in the elementSpec for eventName, so we'll want to change that there too if we change it here]